src/views/ConfigurationVideoView.vue

```diff
@@ -24,6 +28,10 @@
                 density="compact"
                 clearable
               />
+              <v-checkbox
+                v-model="videoStore.enableAutoIceIpFetch"
+                label="Enable auto-retrieval of allowed IP addresses"
```

Comment:
If you add a 'hide-details' to the v-checkbox and to the v-combobox over it you will get less spacey components that will save some valuable vertical space. Check this:

This is your current implementation:
![image](https://github.com/user-attachments/assets/e3f0a9f3-9bd2-46c5-a12d-6b8b67be3f8d)

And this is with the hide-details props on both components:
![image](https://github.com/user-attachments/assets/9c95a6ab-96a5-425c-afc0-505c55009d5f)

I also added a margin bottom to the v-checkbox to harmonize upper and bottom gaps:

````
<div class="flex justify-center flex-col w-[90%] ml-2">
    <v-combobox
       v-model="allowedIceIps"
       multiple
      :items="availableIceIps"
      label="Allowed WebRTC remote IP Addresses"
      class="uri-input"
      variant="outlined"
     chips
      theme="dark"
     density="compact"
      clearable
      **hide-details**
  />
  <v-checkbox
     v-model="videoStore.enableAutoIceIpFetch"
     label="Enable auto-retrieval of allowed IP addresses"
     **hide-details
     class="mb-2"**
   />
</div>
````

Reply:
Nice catch!